react/utils/components.ts

```diff
@@ -54,6 +54,9 @@ export const traverseExtension = (
   isRoot: boolean
 ): AssetEntry[] => {
   const extension = extensions[extensionId]
+  if (!extension) {
+    return []
```

Comment:
this is the heart of the PR, the other changes are related to the updated Extensions type